packs/mssql/actions/execute.query.yaml

```diff
@@ -0,0 +1,33 @@
+---
+pack: mssql
+name: execute.query
+description: Queries MS SQL Server and writes the results to CSV. Returns list of paths to CSV files.
+enabled: true
+
+runner_type: python-script
+entry_point: mssql_runner.py
+
+parameters:
+  action:
+    default: execute_query
+    immutable: true
+    type: string
+  query_string:
+    description: Query to send to MS SQL Server.
+    required: true
+    type: string
+  params:
+    description: Parameters for interpolation in the query_string.
+    type: string
```

Comment:
It would be better for `params` to be an object instead of a string (ideally you should always try to utilize a native type).

This way you don't need to run potentially unsafe `ast.literal_eval` in the code since `params` will already be a dictionary.